packages/patternfly-4/react-table/src/components/Table/EditColumn.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
```

Comment:
I am not well versed on the Table.  Can this not be an experimental component?

Reply:
It should be experimental. But I am confused on what that means for the overall `Table`. Does the whole thing get marked as experimental suddenly, or do we duplicate the whole folder into experimental and have two copies of Table while inline-edit is 'experimental'?

Reply:
How about prefixing relevant props with `experimental`? e.g. in the Table component
```
export type ExperimentalOnRowEdit
experimentalOnRowEdit?: ExperimentalOnRowEdit
```

Reply:
We're looking into flagging the feature as experimental since the Table component itself is already available for production use.

Reply:
Can we also at least mark the associated props related to the feature (eg isEditable) as experimental?

Reply:
Looks like `onRowEdit` is the only one of the new beta/experimental props that's listed in the props table for Table component. I've added the new badges for this prop and the inline edit example - lemme know what you think!